Test pep8-dlk failed

At first, if I run `make pep8-dlk` before `make build`, it fials.
```
$ make pep8-dlk
# Check dlk PEP8
docker run --rm -t blueoil_$(id -un):60d18e0 /bin/bash -c "cd dlk && pycodestyle --ignore=W --max-line-length=120 --exclude='*static/pb*','*docs/*','*.eggs*','*tvm/*','*tests/*','backends/*' ."
Unable to find image 'blueoil_masuda:60d18e0' locally
docker: Error response from daemon: pull access denied for blueoil_masuda, repository does not exist or may require 'docker login'.
See 'docker run --help'.
Makefile:34: recipe for target 'pep8-dlk' failed
make: *** [pep8-dlk] Error 125
```
We need to add `build` dependency into `pep8-dlk` as same as other tests.
And after fixed that, I got the error on the current master branch.
```
$ make pep8-dlk
...
# Check dlk PEP8
docker run --rm -t blueoil_$(id -un):60d18e0-dirty /bin/bash -c "cd dlk && pycodestyle --ignore=W --max-line-length=120 --exclude='*static/pb*','*docs/*','*.eggs*','*tvm/*','*tests/*','backends/*' ."
./python/dlk/core/data_types.py:19:1: E302 expected 2 blank lines, found 1
./python/dlk/core/data_types.py:22:1: E302 expected 2 blank lines, found 1
./python/dlk/core/operators.py:1745:5: E301 expected 1 blank line, found 0
./python/dlk/core/operators.py:1749:1: E302 expected 2 blank lines, found 1
Makefile:34: recipe for target 'pep8-dlk' failed
make: *** [pep8-dlk] Error 1
```
I checked jenkins settings and I found there is not `make pep8-dlk` command.
So, we should do the following tasks to fix them.

- [ ] Add `build` dependency to `pep8-dlk` into `Makefile`
- [ ] Fix codes to follow pep8
- [ ] Add Jenkins settings to test pep8

--- a/Makefile
+++ b/Makefile
@@ -30,7 +30,7 @@ test-dlk: build
 	docker run --rm -t -v /root/.ssh:/root/.ssh --net=host $(IMAGE_NAME):$(BUILD_VERSION) /bin/bash -c "apt-get update && apt-get install -y iputils-ping && cd dlk && python setup.py test"
 
 .PHONY: pep8-dlk
-pep8-dlk:
+pep8-dlk: build
 	# Check dlk PEP8
 	docker run --rm -t $(IMAGE_NAME):$(BUILD_VERSION) /bin/bash -c "cd dlk && pycodestyle --ignore=W --max-line-length=120 --exclude='*static/pb*','*docs/*','*.eggs*','*tvm/*','*tests/*','backends/*' ."
 

--- a/dlk/python/dlk/core/data_types.py
+++ b/dlk/python/dlk/core/data_types.py
@@ -16,8 +16,10 @@
 from typing import List
 import numpy as np
 
+
 def quantized_packed_type(t):
     return f"QuantizedPacked<{t}>"
+
 
 class DataType(object):
 

--- a/dlk/python/dlk/core/operators.py
+++ b/dlk/python/dlk/core/operators.py
@@ -1742,9 +1742,11 @@ class MaxPool(Pool):
     @property
     def is_monotonic(self) -> bool:
         return False
+
     @property
     def preserve_quantization(self) -> bool:
         return True
+
 
 class AveragePool(Pool):
     """Average pooling operator.

